plugin.onedrive/resources/language/english/strings.xml

```diff
@@ -52,4 +52,11 @@
     <string id="30049">Loading %s items in this folder, please wait...</string>
     <string id="30050">Loading %s items from search, please wait...</string>
     <string id="30051">When playing videos, set the subtitle file located next to the video (.srt)</string>
+    <string id="30052">Files</string>
```

Comment:
Add-on string id range is from 32000-33000. Please adjust it when you find the time, and also convert the xmls to .po files.